libs/community/langchain_community/document_loaders/parsers/doc_intelligence.py

```diff
@@ -17,22 +20,42 @@ def __init__(
         api_version: Optional[str] = None,
         api_model: str = "prebuilt-layout",
         mode: str = "markdown",
```

Comment:
We should be using `*` as much as we can to make sure that we dont' break user code accidentally

```suggestion
        mode: str = "markdown",
        *,
```